resources/views/components/about.blade.php

```diff
@@ -1,6 +1,6 @@
 @if ($about !== '')
 <div>
-    <h2 class="text-3xl font-semibold">{{ __('About This Site') }}</h2>
+    <h1 class="text-3xl font-semibold">{{ {{ $title }} }}</h1>
```

Comment:
What's the reason for not wanting to show "About This Site"? Is it weird that the title may say something like "Checkmango Status Page"?

Also, if we do go with this change, we'll need to remove the `en.json` language line for this.

```suggestion
    <h1 class="text-3xl font-semibold">{{ $title }}</h1>
```

Reply:
I do it to make it easier when users search in Google/bing/... "Status of my company", you can say it is for SEO.

On the other hand I did not remove it from the translations because it is used here: https://github.com/cachethq/core/blob/main/src/Filament/Pages/ManageCachet.php